tests/test_crs.py

```diff
@@ -146,3 +146,8 @@ def test_is_valid_false():
 
 def test_is_valid():
     assert is_valid_crs('EPSG:4326')
+
+
+def test_empty_json():
+    with pytest.raises(ValueError):
```

Comment:
@sgillies Ah yeah forgot we had that one.  We raise a `ValueError` if `json.loads()` fails - do we want that to be a `CRSErorr` as well?  It's a slightly different problem, but `CRSError` is derived from `ValueError`, so the change seems appropriate and a little more consistent.